dalston/block-extends/extend-cover-block.css

```diff
@@ -169,81 +173,105 @@
 
 .wp-block-cover.is-style-hover-reveal.dalston-in-viewport.has-background-dim.has-background-dim-0:before,
 .wp-block-cover.is-style-hover-reveal.dalston-in-viewport.has-background-dim.has-background-dim-0.has-background-gradient .wp-block-cover__gradient-background,
+.wp-block-cover.is-style-hover-reveal.dalston-in-viewport .wp-block-cover__background.has-background-dim.has-background-dim-0,
 .wp-block-cover.is-style-hover-reveal.has-background-dim.has-background-dim-0:hover:before,
-.wp-block-cover.is-style-hover-reveal.has-background-dim.has-background-dim-0.has-background-gradient:hover .wp-block-cover__gradient-background {
+.wp-block-cover.is-style-hover-reveal.has-background-dim.has-background-dim-0.has-background-gradient:hover .wp-block-cover__gradient-background,
+.wp-block-cover.is-style-hover-reveal:hover .wp-block-cover__background.has-background-dim.has-background-dim-0 {
```

Comment:
In the current Gutenberg version the `has-background-dim` class is applied to the element `.wp-block-cover__background` and not to the `.wp-block-cover`.